charts/pega/config/deploy/context.xml.tmpl

```diff
@@ -78,6 +78,32 @@
   <Environment name="prconfig/timeout/browser" value="{{ .Env.REQUESTOR_PASSIVATION_TIMEOUT }}" type="java.lang.String" />
   {{ end }}
 
+  {{ if .Env.EXTERNAL_STREAM }}
+  <Environment name="prconfig/services/stream/validate" value="true" type="java.lang.String"/>
```

Comment:
Are these settings required to be in `context.xml.tmpl`? We use to keep platform shipped settings in [prweb.xml](https://github.com/pegasystems/docker-pega-web-ready/blob/master/tomcat-conf/Catalina/localhost/prweb.xml)

Reply:
ok if that's the best practice then I will move these settings to prweb.xml

Reply:
done..changes are present in the next push